src/AccessToken.php

```diff
@@ -82,22 +82,19 @@ public function __construct(
      * accepted.  By default, the id token must have been issued to this OAuth2 client.
      *
      * @param string $token The JSON Web Token to be verified.
-     * @param array $options [optional] {
-     *     Configuration options.
-     *
-     *     @type string $audience The indended recipient of the token.
-     *     @type string $issuer The intended issuer of the token.
-     *     @type string $certsLocation The location (remote or local) from which
+     * @param array $options [optional] Configuration options.
+     * @param string $options.audience The indended recipient of the token.
+     * @param string $options.issuer The intended issuer of the token.
+     * @param string $options.cacheKey The cache key of the cached certs. Defaults to
+     *        the sha1 of $certsLocation if provided, otherwise is set to
+     *        "federated_signon_certs_v3".
+     * @param string $options.certsLocation The location (remote or local) from which
      *        to retrieve certificates, if not cached. This value should only be
      *        provided in limited circumstances in which you are sure of the
      *        behavior.
-     *     @type string $cacheKey The cache key of the cached certs. Defaults to
-     *        the sha1 of $certsLocation if provided, otherwise is set to
-     *        "federated_signon_certs_v3".
-     *     @type bool $throwException Whether the function should throw an
+     * @param bool $options.throwException Whether the function should throw an
```

Comment:
This is different than the style we use for `google/cloud`. Are we moving to this format over there as well?

Reply:
The documentation generator used doesn't support the style we use on google cloud php. I wouldn't be opposed to moving to this style there as well, but it's a bigger job and would require changes to the generated code as well.